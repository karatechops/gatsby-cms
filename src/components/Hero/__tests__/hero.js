import React from 'react';
import renderer from 'react-test-renderer';

import Hero from '../index';

describe('Hero', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Hero
          title="Starter"
          desc="My description."
          bg="https://test.com/iimage.jpg"
        />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
