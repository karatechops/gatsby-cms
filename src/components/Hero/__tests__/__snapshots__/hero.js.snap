// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero renders correctly 1`] = `
<div
  className="StyledBox-sc-13pk1d4-0 bRIVSP"
>
  <div
    className="StyledBox-sc-13pk1d4-0 iaNxlk"
  >
    <h1
      className="StyledHeading-sc-1rdh4aw-0 iMVrbt"
      size="large"
    >
      Starter
    </h1>
    <div
      className="StyledBox__StyledBoxGap-sc-13pk1d4-1 irlfVP"
    />
    <span
      className="StyledText-sc-1sadyjn-0 brldlG"
    >
      My description.
    </span>
    <div
      className="StyledBox__StyledBoxGap-sc-13pk1d4-1 irlfVP"
    />
    <div
      className="StyledBox-sc-13pk1d4-0 ffeAql"
    >
      <button
        className="StyledButton-sc-323bzc-0 aNlbA"
        onBlur={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <span
          className="StyledText-sc-1sadyjn-0 bnqisW"
        >
          Learn more
        </span>
      </button>
      <div
        className="StyledBox__StyledBoxGap-sc-13pk1d4-1 iChEkS"
      />
      <button
        className="StyledButton-sc-323bzc-0 kHrfdt"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseOut={[Function]}
        onMouseOver={[Function]}
        type="button"
      >
        <span
          className="StyledText-sc-1sadyjn-0 bLEHmJ"
        >
          Login
        </span>
      </button>
    </div>
  </div>
</div>
`;
